src/patternfly/components/Pagination/pagination.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../patternfly-imports";
+
+.pf-c-pagination {
+  // Base
+  --pf-c-pagination--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-pagination--child--MarginRight: var(--pf-global--spacer--lg);
+
+  // Dropdown
+  --pf-c-pagination__menu-text--Color: var(--pf-global--Color--200);
+
+  // Nav
+  --pf-c-pagination__nav--c-button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-pagination__nav--c-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-pagination__nav--c-button--FontSize: var(--pf-global--FontSize--md);
+
+  // Nav page select
+  --pf-c-pagination__nav-page-select--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page-select--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page-select--child--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-pagination__nav-page-select--c-form-control--MaxWidth: #{pf-size-prem(48px)}; // custom value to show 2 numbers
+
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-end;
```

Comment:
do you need this id you have line 39?

Reply:
oh good catch, that's left over from something that changed